wx-base/src/generated.rs

```diff
@@ -140,19 +144,19 @@ pub mod methods {
 
     // wxEvent
     pub trait EventMethods: ObjectMethods {
-        fn clone(&self) -> EventIsOwned<false> {
-            unsafe { EventIsOwned::from_ptr(ffi::wxEvent_Clone(self.as_ptr())) }
+        fn clone(&self) -> WeakRef<Event> {
+            unsafe { WeakRef::<Event>::from(ffi::wxEvent_Clone(self.as_ptr())) }
```

Comment:
We should return cloned object as Event instead of WeakRef<Event>, but I'll work on other PR/Issue.